contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -88,6 +96,179 @@ contract AutomationRegistry2_3_CheckUpkeep is AutomationRegistry2_3_SetUp {
   }
 }
 
+contract AutomationRegistry2_3_Withdraw is AutomationRegistry2_3_SetUp {
+  address internal aMockAddress = address(0x1111111111111111111111111111111111111113);
+
+  function getLinkBalance(address balanceAddress) public view returns (uint256) {
+    return link.balanceOf(balanceAddress);
+  }
+
+  function getNonLinkBalance(address balanceAddress) public view returns (uint256) {
+    return nonLink.balanceOf(balanceAddress);
+  }
```

Comment:
nitpick, but are these helper functions really necessary? `getLinkBalance()` is the same number of characters as `link.balanceOf()` lol and it doesn't perform any extra function. I feel like the native balance getter functions are fine.